Adjustments on CMakeLists.txt to properly install some modules

bjmarfito opened the pull request:
- Adjustments on CMakeLists.txt to properly install ISSI and filter modules
- Make some adjustments on Stack.py

bjmarfito committed:
Updated CMakeLists.txt to properly install the ISSI files thru CMake

bjmarfito committed:
Update CMakeLists.txt to compile the src to generate filter.so

bjmarfito committed:
Adjusted spaces on the code

bjmarfito committed:
Update Stack.py

rtburns-jpl commented on components/isceobj/Filter/CMakeLists.txt:
```diff
@@ -1,4 +1,16 @@
+Python_add_library(filter MODULE
+	bindings/filtermodule.cpp
+	src/cpxPhaseFilter.cpp
+	src/EdgeFilter.cpp
+	src/Filter.cpp
+	src/filterPhase.cpp
+	src/GaussianFilter.cpp
+	src/MeanFilter.cpp
+	src/medianFilter.cpp
+        )
```
Please use 4 spaces of indentation here

bjmarfito committed:
Updated CMakeLists.txt for compiling the filter codes

bjmarfito committed:
Update CMakeLists.txt

bjmarfito committed:
Delete CMakeLists.txt

bjmarfito committed:
Update CMakeLists.txt to install ISSI modules

bjmarfito committed:
Add files via upload

bjmarfito commented:
Pls. check @rtburns-jpl if I addressed your suggestions. I've made some changes on the CMakeLists.txt regarding the installation of ISSI.py to the applications folder.

bjmarfito committed:
Update CMakeLists.txt

bjmarfito commented:
@rtburns-jpl, just added it now. Apologies, I've overlooked it.

bjmarfito commented:
Yes, @rtburns-jpl. Your suggestions worked.

rtburns-jpl commented:
Great! I think you can also remove contrib/issi/applications/CMakeLists.txt and contrib/issi/components/CMakeLists.txt as they are no longer used.

bjmarfito committed:
Delete CMakeLists.txt

bjmarfito committed:
Delete CMakeLists.txt

bjmarfito commented:
I've deleted those CMakeLists.txt files @rtburns-jpl. Thank you for your suggestions.

rtburns-jpl reviewed (approved):
Looks good, thank you!
